qa/pull-tester/rpc-tests.py

```diff
@@ -194,6 +195,12 @@ def main():
             # (for parallel running efficiency). This combined list will is no
             # longer sorted.
 
+    # Remove the test cases that the user has explicitly asked to exclude.
+    if args.exclude:
+        [ test_list.remove(exclude_test + ".py")
```

Comment:
why use [ ] here?

I think this would be more clear:
```
    if args.exclude:
        for exclude_test in args.exclude.split(','):
            if exclude_test + ".py" in test_list:
                test_list.remove(exclude_test + ".py")
```
And it's the same number of lines.